pkg/task/backup.go

```diff
@@ -56,6 +59,7 @@ func DefineBackupFlags(flags *pflag.FlagSet) {
 		" use for incremental backup, support TSO only")
 	flags.String(flagBackupTS, "", "the backup ts support TSO or datetime,"+
 		" e.g. '400036290571534337', '2018-05-11 01:42:23'")
+	flags.Int64(flagGCTTL, 0, "the TTL that pd hold for this br safePoint")
```

Comment:
it's hard to digest what this means.

also, can you set the default to `DefaultBRGCSafePointTTL` here instead of 0 to remove the special case in `GetTTL()`? (perhaps change the flag type to Duration too)

Reply:
okay, I'll change it.